crates/extrinsics/src/env_check.rs

```diff
@@ -287,7 +287,7 @@ mod tests {
                         vec!["i32"].into_iter().map(AsRef::as_ref),
                     ))
                     .done()])
-                .returns(ReturnTypeSpec::new(None))
+                .returns(ReturnTypeSpec::new(TypeSpec::default()))
```

Comment:
The default value is: 
```rust
ReturnTypeSpec::new(
    TypeSpec::with_name_str::<ink_primitives::ConstructorResult<()>>("ink_primitives::ConstructorResult")
)
```

Not sure if `ink_primitives` is imported.

Reply:
It is not imported; that is why I used the default.